week3/homework/src/index.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.get('/todo/:id', (request, response) => {
+  fs.readFile('./todos.json', 'utf8', (error, todo) => {
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('no data found');
+        response.status(404).send('');
+      } else console.error(error);
+    } else {
+      const newList = todo.split('\n');
+      const id = request.params.id;
+      response.json({ result: newList[id] });
+    }
+  });
+});
+
+app.delete('/todo/:id', (request, response) => {
+  const id = request.params.id;
+  fs.readFile('./todos.json', 'utf8', (error, todosBinary) => {
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('no data found');
+        response.status(404).send('');
+      } else console.error(error);
+    } else {
+      const todos = JSON.parse(todosBinary);
+      todos.splice(id - 1, 1);
+
+      fs.writeFile('./todos.json', JSON.stringify(todos), error => {
```

Comment:
The `error` also here should be checked, so you send the response to user accordingly. Like what you did in the previous routes.

Reply:
Done